source/PyMaterialX/PyDefinition.cpp

```diff
@@ -37,7 +42,12 @@ void bindPyDefinition(py::module& mod)
         .def("setLanguage", &mx::Implementation::setLanguage)
         .def("hasLanguage", &mx::Implementation::hasLanguage)
         .def("getLanguage", &mx::Implementation::getLanguage)
-        .def_readonly_static("CATEGORY", &mx::Implementation::CATEGORY);
+        .def_readonly_static("CATEGORY", &mx::Implementation::CATEGORY)
+        .def_readonly_static("CATEGORY", &mx::Implementation::NODE_DEF_ATTRIBUTE)
```

Comment:
I think these need to be assigned their unique strings, though?  For example, this one should have the string "NODE_DEF_ATTRIBUTE" rather than "CATEGORY".